packages/core/src/html/CustomListIconProcessor.ts

```diff
@@ -98,6 +104,8 @@ IconAttributes | null => {
     height: node.attribs['i-height'] !== undefined ? node.attribs['i-height'] : iconAttrsSoFar?.height,
     size: node.attribs['i-size'] !== undefined ? node.attribs['i-size'] : iconAttrsSoFar?.size,
     className: node.attribs['i-class'] !== undefined ? node.attribs['i-class'] : iconAttrsSoFar?.className,
+    oneOff: (node.attribs['i-one-off'] === true || node.attribs['i-one-off'] === 'true')
+      ? true : iconAttrsSoFar?.oneOff,
```

Comment:
```suggestion
    oneOff: node.attribs['i-one-off'] !== undefined 
    ? (node.attribs['i-one-off'] === true || node.attribs['i-one-off'] === 'true') 
    : iconAttrsSoFar?.oneOff,
```
**But is this oneOff value supposed to be inherited?** 
Also, if a user inputs false into the attribute, but the `iconAttrsSoFar?.oneOff` is true, the output here will be true.

Reply:
Its coded such that `iconAttrsSoFar?.oneOff` will never be true. Since if it was then it would'nt get saved into iconAttrsSoFar.

Reply:
?? Then why do we need this check?